lib/scxtest/scx_test_map.c

```diff
@@ -26,14 +26,14 @@ struct scx_percpu_map_entry {
 	struct scx_percpu_test_map *map;
 };
 
-static struct scx_map_entry *scx_map_entries = NULL;
-static int scx_map_entries_count = 0;
+static __thread struct scx_map_entry *scx_map_entries = NULL;
```

Comment:
Maybe a silly question but do all of these need to be thread local?

Reply:
I'm not super sure and haven't reviewed this code well. My theory is that it's better to play it safe and start with everything thread local then relax it later or for new additions, given none of this code was written with multithreading in mind.